src/components/calculator/calculator.scss

```diff
@@ -0,0 +1,73 @@
+$color-white: #fff;
+$color-indigo: #7d3c98;
+$color-black: #566573;
+$background-clr: #f4f8fa;
+$light-orange-clr: #e59866;
+$color-primary: #3498db;
+$orange-clr: #ff7800;
+$light-orange: #dc7633;
+$black: #130f40;
+$light-clr: #666;
+$dark-orange: #a04000;
+$dark-red: #c0392b;
+$box-shadow: 0 0.5rem 1.5rem rgba(0, 0, 0, 0.1);
+$border: 0.2rem solid rgba(0, 0, 0, 0.1);
+$outline: 0.1rem solid rgba(0, 0, 0, 0.1);
+$outline-hover: 0.2rem solid var(--black);
+
+.calculator-container {
+  margin: 8rem 0 6rem;
+  border-radius: 1rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 1rem;
+
+  .wrapper {
+    width: 50%;
+    padding: 2rem 0;
+    background: $background-clr;
+    display: flex;
+    flex-direction: column;
+    -webkit-box-shadow: 3px 7px 5px 0 rgba(50, 50, 50, 0.75);
+    -moz-box-shadow: 3px 7px 5px 0 rgba(50, 50, 50, 0.75);
+    box-shadow: 3px 2px 5px 0 rgba(50, 50, 50, 0.75);
+    border-radius: 0.25rem;
+
+    .display-screen {
+      background: $color-white;
+      height: 5rem;
+      margin: 0 4rem 2rem 4rem;
+      display: flex;
+      align-items: center;
+      justify-content: flex-end;
+      padding-right: 1rem;
+      font-size: 1.25rem;
+    }
```

Comment:
- [ ] Good job with developing the calculator component , but as mentioned in the project we have to match this design: 
![image](https://user-images.githubusercontent.com/97521557/211631803-4e583bf3-c3d5-4dfe-ae4a-b878d86d3796.png)

Our current design is a bit defferent see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/211631922-36b553d8-2b36-44f4-bdb1-c38d4846afdb.png)

Can we try to make the design match as much as possible :+1: